packages/app/src/app/pages/Dashboard/Content/routes/SyncedSandboxes/index.tsx

```diff
@@ -78,7 +78,7 @@ export const SyncedSandboxesPage = () => {
       items={itemsToShow}
     >
       <Helmet>
-        <title>{param || 'Dashboard'} - CodeSandbox</title>
+        <title>{param || 'Synced sandboxes'} - CodeSandbox</title>
```

Comment:
Do these still exist @alexnm @olarclara?

Reply:
@tristandubbeld yep, the CodeSandbox team has some synced sandboxes.